src/components/HdResponsive.vue

```diff
@@ -18,42 +16,46 @@ export default {
     return {
       matches: {},
       indeterminate: !matchMediaAvailable,
+      listeners: {},
     };
   },
   watch: {
     breakpoints() {
       this.setResponsiveFlags();
     },
   },
-  created() {
+  beforeMount() {
     this.setResponsiveFlags();
   },
-  mounted() {
-    this.addResizeEvents();
-  },
-  beforeDestroy() {
-    this.removeResizeEvents();
+  created() {
+    this.setResponsiveFlags();
```

Comment:
Any special reason for calling it in `beforeMount` and `created`?

Reply:
I removed the _created_ one. It will come up with my next push